www/package.json

```diff
@@ -32,6 +32,7 @@
     "clsx": "^1.1.1",
     "cssnano": "^5.1.13",
     "docusaurus-preset-shiki-twoslash": "^1.1.38",
+    "prism-react-renderer": "^1.3.5",
```

Comment:
are we sure this doesn't negatively affect page load times? would be great to use `shiki` if possible - no need to maintain two different syntax highlighter dependencies, themes etc.